### PR TITLE
Add debts on the dashboard

### PR DESCRIPTION
**Pull request by djaiss**
This PR adds support for displaying debts on the dashboard, regardless of the contacts, in order to have a complete overview.

image

#### Code-related tasks
- [x] Tests added for this feature/bug.
- [x] Impact on the seeders.
- [x] Impact on the API.

#### If the code changes the SQL schema
- [x] Impact on account export.
- [x] Impact on importing data with `vCard` and `.csv` files.
- [x] Impact on account reset and deletion.

#### Other tasks
- [x] [CHANGELOG](https://github.com/monicahq/monica/blob/master/CHANGELOG) entry added, if necessary, under `UNRELEASED`.

### CODE REVIEW

File: resources/assets/js/app.js [hunk 1]
```diff
@@ -174,6 +174,14 @@ import axios from 'axios';
 import VueI18n from 'vue-i18n';
 Vue.use(VueI18n);
 
+// Moments
+import moment from 'moment';
+Vue.filter('formatDate', function(value) {
+    if (value) {
+        return moment(String(value)).format('LL')
```
Comment (asbiin) [inline resources/assets/js/app.js]: Great, but it must support locale. Just add `moment.locale(lang);` on line 210 after `loadLanguageAsync(window.Laravel.locale, true).then((lang) =>`

Comment (djaiss) [reply resources/assets/js/app.js]: Good point. I was lazy. Thanks.

Comment (djaiss) [reply resources/assets/js/app.js]: I've added it and tested it, it seems to work. Can you double check?

Comment (djaiss) [reply resources/assets/js/app.js]: @asbiin waiting for your validation here!

Comment (asbiin) [reply resources/assets/js/app.js]: Seems good!

**Comment by asbiin** [comment]
What about this ?
"Perhaps it should be more usefull to see the total for 1 person instead of all details?"

**Comment by djaiss** [comment]
> "Perhaps it should be more usefull to see the total for 1 person instead of all details?"

Yes, but it's also more work :-D But yeah, it would be interesting. But we can ship this as is, then improve it in another PR.

**Pull request merged**